src/components/TracksTable.vue

```diff
@@ -202,6 +210,31 @@ export default {
     reloadSelected() {
       this.selected = this.selected.map((s) => this.tracksObj[s.id]);
     },
+    sortFunction(items, sortBy, sortDesc) {
+      let sortFunction = null;
+      switch (sortBy[0]) {
+        case "album_id":
+          sortFunction = compareTracks;
+          break;
+        case "genre_ids":
+          sortFunction = compareTracksByGenre;
+          break;
+        case "length":
+          sortFunction = (t1, t2) => t1.length - t2.length;
+          break;
+        case "number":
+          sortFunction = (t1, t2) => t1.number - t2.number;
+          break;
+        case "title":
+          sortFunction = (t1, t2) =>
+            compareStrings(t1.normalized_title, t2.normalized_title);
+          break;
+        case "track_artists":
+          sortFunction = compareTracksByArtist;
+      }
+      const sorted = sortFunction ? items.sort(sortFunction) : items;
+      return sortDesc[0] ? sorted.reverse() : sorted;
+    },
```

Comment:
I'm not sure about all these options. In general, I would prefer to only add features we have a clear use case for. Sorting by number for example doesn't seem like it would ever be useful.

Reply:
We sort by number in the album view.
Moreover, if we keep sorting inside the table headers I wouldn't restrict these because we don't see the use. It feels weridly restrictive to only allow certain properties. If a user wants to sort all their tracks by number (because they want to know which tracksnr in their library is the highest or whatever) they should be able to